Makefile

```diff
@@ -54,7 +54,8 @@ RM_F = $(RM) -f
 RM_RF = $(RM) -rf
 
 RUNTIME ?= dotnet
-VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
+VERSION = $(shell head -n1 VERSION)
```

Comment:
Should this be `VERSION ?=` so it can get overridden from the command line?

Reply:
It should be `VERSION_GIT ?=`. The naming of the variables is not good for this use case.

Would you prefer renaming `VERSION_GIT` to `NEW_VERSION` with `make version NEW_VERSION=...` or renaming `VERSION_GIT` back to `VERSION` with `make version VERSION=...`?

Reply:
How about `VERSION` for the git/manually defined version, and `CURRENT_VERSION` for the one read from the file?

Reply:
> How about `VERSION` for the git/manually defined version, and `CURRENT_VERSION` for the one read from the file?

OK

Reply:
> Should this be `VERSION ?=` so it can get overridden from the command line?

Correction: The variable can be overridden from the command line without `?=`. Only difference between `=` and `?=` is that `=` ignores overrides with an empty string.